src/stream/src/executor/dispatch.rs

```diff
@@ -228,14 +180,23 @@ impl DispatchExecutorInner {
                         dispatcher.dispatch_data(chunk.clone()).await?;
                     }
                 }
+                self.metrics
+                    .actor_output_buffer_blocking_duration_ns
+                    .with_label_values(&[&self.actor_id_str])
+                    .inc_by(start_time.elapsed().as_nanos() as u64);
```

Comment:
So it will not only count the blocking time but also the normal dispatch time?

Reply:
If there's no back pressure, the time should be close to 0.

Reply:
Have you benched this?

Reply:
I think you also refactored the Grafana dashboard in this PR, so maybe this PR can be merged after my PR is merged?

Reply:
> If there is actually no backpressure and resource is over-provision, but the metrics show that there is, the metrics can be pretty misleading.

Is there any workload without back-pressure?